tutorials/scripting/gdscript/warnings/unassigned_variable.rst

```diff
@@ -0,0 +1,36 @@
+UNASSIGNED_VARIABLE
+=======================
+
+The warning message is:
+
+.. code-block:: none
+
+    The variable "my_var" is used before being assigned a value.
+
+The default warning level for this warning is **Warn**.
+To modify it, see :ref:`ProjectSettings.debug/gdscript/warnings/unassigned_variable<class_ProjectSettings_property_debug/gdscript/warnings/unassigned_variable>`.
+
+When this warning occurs
+------------------------
+
+This warning may appear when attempting to use a variable that hasn't had a value assigned to it yet.
```

Comment:
May be worth mentioning that in Godot an unassigned variable is essentially `null`